<launch>
  <arg name="enalbe_loop" default="true"/>
  <arg name="input_topic_start_pose" default="/initialpose3d"/>
  <arg name="input_topic_end_pose" default="/planning/mission_planning/goal"/>
  <arg name="input_topic_checkpoint" default="/planning/mission_planning/checkpoint"/>
  <arg name="input_topic_autoware" default="/autoware/state"/>
  <arg name="input_topic_loop_limit" default="/planning/mission_planning/loop_limit"/>
  <arg name="output_topic_checkpoint" default="/planning/mission_planning/checkpoint"/>
  <arg name="output_topic_goal" default="/planning/mission_planning/goal"/>
  <arg name="output_topic_engage" default="/autoware/engage"/>

  <group if="$(var enalbe_loop)">
    <node pkg="loop_route" exec="loop_route" name="loop_route" output="screen">
      <remap from="input_start_pose" to="$(var input_topic_start_pose)"/>
      <remap from="input_end_pose" to="$(var input_topic_end_pose)"/>
      <remap from="input_checkpoint" to="$(var input_topic_checkpoint)"/>
      <remap from="input_autoware_state" to="$(var input_topic_autoware)"/>
      <remap from="input_limit" to="$(var input_topic_loop_limit)"/>
      <remap from="output_checkpoint" to="$(var output_topic_checkpoint)"/>
      <remap from="output_goal" to="$(var output_topic_goal)"/>
      <remap from="output_engage" to="$(var output_topic_engage)"/>
    </node>
  </group>
</launch>